src/components/About/index.tsx

```diff
@@ -0,0 +1,12 @@
+import './style.scss';
+
+const About: React.FC = () => {
+  return (
+    <div id='about' className='about'>
```

Comment:
why give same `id` & `className`?

Reply:
id for redirecting the user on the click of nav buttons.